apps/back-office/translations/nl.json

```diff
@@ -53,14 +53,23 @@
     "title": "Wijzig status melding {publicId}",
     "label": "Wijzig status",
     "options": {
-      "default": "Kies een status",
-      "processing": "In behandeling",
-      "completed": "Afgehandeld"
+      "default": "Kies een status"
     },
     "submit-button": "Wijzig status",
     "errors": {
       "invalid-state": "Ongeldige status",
       "melding-not-found": "An error occurred"
     }
+  },
+  "generic": {
+    "melding-state": {
```

Comment:
I don't think we need the 'melding-' part, pretty much everything here is about a melding.